frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -47,18 +50,29 @@ export const Header = styled.header`
   flex-direction: column;
 `;
 
-export const TitleAndDescriptionContainer = styled.div`
-  padding: 0.5rem 0.5rem 0 0.5rem;
+export const TitleAndDescriptionContainer = styled(FullWidthContainer)`
+  margin-top: 0.5rem;
 
   ${breakpointMinSmall} {
-    padding: 1rem 1rem 0 1rem;
+    margin-top: 1rem;
   }
 
   ${breakpointMinLarge} {
-    padding: 1.5rem 1.5rem 0 1.5rem;
+    margin-top: 1.5rem;
   }
 `;
 
+export const DashboardTabsContainer = styled(FullWidthContainer)`
+  ${breakpointMaxSmall} {
+    padding-left: 0;
+    padding-right: 0;
+  }
+`;
```

Comment:
Adjust public/embed dashboard tabs styling to be consistent with full-app dashboards (see discussion for context: https://metaboat.slack.com/archives/C064QMXEV9N/p1707542235356899?thread_ts=1707503415.376959&cid=C064QMXEV9N)